chrysom/README.md

```diff
@@ -0,0 +1,36 @@
+# chrysom
+The client library for communicating with Argus over HTTP.
+[![PkgGoDev](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)
+
+## Summary
+This package enables CRUD operations with Argus: 
+
+### CRUD Operations
+
+Gets all items that belong to a given owner
+```
+func (c *Client) GetItems(ctx context.Context, owner string) (Items, error)
+```
```

Comment:
I would suggest changing each of these to a bullet in the list and then describe what happens when the function is called:

```suggestion
- On a `GetItems()` call, chrysom returns a list of all items belonging to the provided owner.
```

In general, we want our readme to have full sentences.